docs/data/stashcache/install-origin.md

```diff
@@ -11,108 +11,134 @@ to export its data to the StashCache data federation.
+  only the `xrootd` user is utilized for the running daemons.
+* __Host certificate:__ The origin service uses a host certificate to authenticate with the caches it serves.
   The [host certificate documentation](/security/host-certs.md) provides more information on setting up host
   certificates.
-* __Network ports:__ The StashCache Origin service requires the following ports open:
-    * Inbound TCP port 1094 for file access via the XRootD protocol
-    * Outbound TCP port 1213 to `redirector.osgstorage.org` for connecting to the data federation
-    * Outbound UDP port 9619 for reporting to `collector1.opensciencegrid.org` and `collector2.opensciencegrid.org`
-* __Hardware requirements:__ We recommend that a StashCache origin has at least 1Gbps connectivity and 8GB of
-  RAM.  We suggest that several gigabytes of local disk space be available for log files.
+* __Network ports:__ The origin service requires the following ports open:
+  * Inbound TCP port 1094 for file access via the XRootD protocol
+  * Outbound TCP port 1213 to `redirector.osgstorage.org` for connecting to the data federation
+  * Outbound UDP port 9930 for reporting to `xrd-report.osgstorage.org` and `xrd-mon.osgstorage.org` for monitoring.
+* __Hardware requirements:__ We recommend that an origin has at least 1Gbps connectivity and 8GB of RAM.
+  We suggest that several gigabytes of local disk space be available for log files,
+  although some logging verbosity can be reduced.
 
 As with all OSG software installations, there are some one-time steps to prepare in advance:
 
-* Ensure the host has [a supported operating system](/release/supported_platforms.md).
-  A RHEL 7-based operating system is strongly recommended.
 * Obtain root access to the host
 * Prepare [the required Yum repositories](/common/yum.md)
 * Install [CA certificates](/common/ca.md)
 
-Installing the StashCache origin
---------------------------------
+Installing the Origin
+---------------------
 
-The StashCache daemon consists of an XRootD server and an HTCondor-based service for collecting and reporting
-monitoring data about the origin. To simplify installation, OSG provides convenience RPMs that install all required
+The origin service consists of one or more XRootD daemons and their dependencies for the authentication infrastructure.
+To simplify installation, OSG provides convenience RPMs that install all required
 software with a single command:
 
 ```console
-root@host # yum install stashcache-origin-server
+root@host # yum install --enablerepo=osg-testing stash-origin
 ```
 
-For this installation guide, we assume that the data to be exported to the federation is mounted at `/stash`
+!!! note
+    This document covers a completely overhauled stash origin service based on XCache 1.0.2;
+    while this has not yet been released into the OSG production repository,
+    we strongly recommend using this version for configuration and accounting improvements.
```

Comment:
We should be able to get rid of this note upon release

Reply:
Done. Made corresponding change in install-cache.md.